eth/db/journal.py

```diff
@@ -247,7 +260,9 @@ def __getitem__(self, key: bytes) -> bytes:
 
         val = self.journal[key]
         if val is DELETED_ENTRY:
-            raise KeyError(key, "item was explicitly deleted in JournalDB")
+            raise KeyError(key, "item to be deleted from wrapped DB, by JournalDB")
```

Comment:
nit: maybe something like "this item's deletion has been journaled" or "this item is pending deletion"